docs/Contributing.asciidoc

```diff
@@ -325,12 +326,19 @@ workers need to be configured to connect to the main web UI port (add `HOST = ht
 * Add 3rd party JavaScript and CSS file to `assets/assetpack.def`. When restarting
   the web server the new/updated files are pulled automatically. Also take care to
   <<Contributing.asciidoc#update-asset-cache,update the asset cache for the openSUSE RPM package>>.
-* Other dependencies need to be added to `openQA.spec` or `os-autoinst.spec`.
-* Perl dependencies need to be added *additionally* to `cpanfile`.
 * To easily get all necessary dependencies on openSUSE you can install the
   package `openQA-devel`. In other cases one can rely on the `cpanfile` and
   read out the dependencies from the spec file for the rest.
 
+=== Adding dependencies
+
+In openQA, there is a `dependencies.yaml` file including a list of dependencies,
+seperated in groups.  Add a new dependency there and run `make update-deps`.
```

Comment:
a double whitespace

Reply:
Ok, fixed, although spaces are not significant in asciidoc, and such spaces can appear if one uses vim to format a number of lines with `gq`.